modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryRelatedObjectsResourceTest.java

```diff
@@ -1044,24 +1120,29 @@ private void _testPostCustomObjectEntryWithInvalidNestedSystemObjectEntries(
 		UserAccount userAccount = _randomUserAccount();
 
 		JSONObject jsonObject = HTTPTestUtil.invoke(
-			_toBody(manyToOne, objectRelationship, userAccount),
+			_toBody(null, manyToOne, objectRelationship, userAccount),
 			_objectDefinition1.getRESTContextPath(), Http.Method.POST);
 
 		Assert.assertEquals("BAD_REQUEST", jsonObject.get("status"));
 	}
 
 	private void _testPostCustomObjectEntryWithNestedSystemObjectEntry(
-			ObjectRelationship objectRelationship, boolean manyToOne)
+			String[] customObjectField, ObjectRelationship objectRelationship,
+			boolean manyToOne)
 		throws Exception {
 
+		JaxRsApplicationDescriptor jaxRsApplicationDescriptor =
```

Comment:
This `JaxRsApplicationDescriptor` is being used 24 lines after, please, sort the code